### PR TITLE
Added Grafana

### PR DESCRIPTION
**Pull request by HammyHavoc**
Lovely software.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -662,6 +662,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
   * [Dashing](http://shopify.github.io/dashing/) - Sinatra based, open sourced dashboard builder from Shopify. ([Demo](http://dashingdemo.herokuapp.com/sample), [Source Code](https://github.com/Shopify/dashing)) `MIT` `Ruby`
   * [Dj Diabetes](https://github.com/foxmask/dj-diabetes) - My Glucose Manager - follow your daily health  `BSD` `Python`
   * [Freeboard](https://freeboard.io) - An open source real-time dashboard builder for IOT and other web mashups. ([Source Code](https://github.com/freeboard/freeboard)) `MIT` `Nodejs`
+  * * [Freeboard](http://grafana.org/) - An open source dashboard for data visualization. ([Source Code](https://github.com/grafana)) `Go` `Nodejs`
```
Comment (nodiscc) [inline README.md]: * wrong naming :)
 * there's one extra ` *`

**Comment by HammyHavoc** [comment]
Thanks for catching that bug! (- :+1:

**Comment by AndyR207** [comment]
Please remove one language or concatenate them using a `/`, missing a language.

**Comment by HammyHavoc** [comment]
Removed a language and added the license.

**Comment by Kickball** [comment]
Looks good, thanks for the pull request!

**Pull request merged**